src/redux/sagas/expenditures/cancelDraftExpenditure.ts

```diff
@@ -70,8 +176,6 @@ function* cancelDraftExpenditure({
     );
   }
 
```

Comment:
I think you still need to close the channels, have a look at `createExpenditure` to see how to do that with mulitple channels.